app/views/matches/replay.slim

```diff
@@ -1,4 +1,5 @@
 = javascript_include_tag "page_specific/match_replay.js"
+= javascript_include_tag "page_specific/underscore-min.js"
```

Comment:
I totally thought that was a built in function I did not know about. I'll go ahead and merge this while we talk about a better solution for the entire site :)